ocmonitor/cli.py

```
@@ -1153,8 +1314,12 @@ def metrics(ctx, port, host):
         return
 
     try:
-        console.print(f"[status.success]Starting Prometheus metrics server...[/status.success]")
-        console.print(f"[metric.label]Endpoint:[/metric.label] [metric.value]http://{host}:{port}/metrics[/metric.value]")
+        console.print(
+            f"[status.success]Starting Prometheus metrics server...[/status.success]"
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)